doc/Motr_Configuration.rst

```diff
@@ -0,0 +1,74 @@
+==========
```

Comment:
Where is the original document? Where is the document referenced?

I don't think we need this document any longer.

Reply:
Link to the document: [Source Document](https://seagatetechnology-my.sharepoint.com/:w:/r/personal/valery_vorotyntsev_seagate_com/_layouts/15/Doc.aspx?sourcedoc=%7BC7B2AC10-5EB5-442E-836E-CE6530382B6E%7D&file=Colibri%20Configuration%201-pager.docx&action=default&mobileredirect=true)

Reply:
If you confirm, I can close this PR.

Reply:
Please mark this document as "not needed". It is out-dated, and if it exists, it will be misleading.